checker/jdk/nullness/src/java/util/Collection.java

```diff
@@ -323,7 +323,7 @@
      * @throws UnsupportedOperationException if the <tt>remove</tt> operation
      *         is not supported by this collection
      */
-    boolean remove(@Nullable Object o);
+    boolean remove(Object o);
 
 
     // Bulk Operations
```

Comment:
We'd probably want for `containsAll`, `removeAll`, and `retainAll` to require `Collection<? extends Object>` (or `Collection<? extends @NonNull Object>` if it's necessary to be explicit here, but I don't think so?).

That also goes for the redeclarations on `List` and `Set`.

Reply:
Great point!  I overlooked those methods.  Thanks for pointing them out.